benchmark/run.js

```diff
@@ -10,16 +10,26 @@ const cli = CLI(`usage: ./node run.js [options] [--] <category> ...
 
   --filter pattern          string to filter benchmark scripts
   --set    variable=value   set benchmark variable (can be repeated)
+  --format simple/csv       optional value that specifies the output format
 `, {
   arrayArgs: ['set']
 });
 const benchmarks = cli.benchmarks();
+const validFormats = ['csv', 'default'];
```

Comment:
It could also be removed :))), but I wanted to add something to not let 'csv' alone :D